### PR TITLE
Improve text grammar

### PR DESCRIPTION
**Pull request by rhumbertgz**
- Missing commas 
- Pronoun problems

### CODE REVIEW

File: _tour/traits.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ trait Iterator[A] {
 }
 ```
 
-Extending the `trait Iterator[A]` requires a type `A` and implementations of the methods `hasNext` and `next`.
+Extending the `trait Iterator[A]` requires a type `A`, and implementations of the methods `hasNext` and `next`.
```
Comment (SethTisue) [inline _tour/traits.md]: adding a comma here is incorrect. a comma would only be correct here if it introduced an independent clause

if you feel the sentence is hard to read without the comma, we need to find some other solution.

Comment (rhumbertgz) [reply _tour/traits.md]: No problem, maybe it just my opinion 👍🏻. 
I close this PR

Comment (SethTisue) [reply _tour/traits.md]: but I liked the other two changes!

Comment (rhumbertgz) [reply _tour/traits.md]: @SethTisue I'll remove the conflicting change and leave the other

Comment (rhumbertgz) [reply _tour/traits.md]: @SethTisue this is the new PR[#1665](https://github.com/scala/docs.scala-lang/pull/1665)

**Pull request closed without merging**